game/test/contrib/utils/components/skill/SkillTest.java

```diff
@@ -0,0 +1,62 @@
+package contrib.utils.components.skill;
+
+import static org.junit.Assert.*;
+
+import core.Entity;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.function.Consumer;
+
+public class SkillTest {
+
+    private static int value = 0;
+    private final int baseCoolDownInSeconds = 5;
+
+    private Entity entity;
+    private Skill skill;
+    private Consumer<Entity> skillFunction = entity -> value++;
+
+    @Before
+    public void setup() {
+        entity = new Entity();
+        skill = new Skill(skillFunction, baseCoolDownInSeconds);
+    }
+
+    @After
+    public void cleanup() {
+        value = 0;
+    }
+
+    @Test
+    public void execute() {
+        assertTrue(skill.canBeUsedAgain());
+        skill.execute(entity);
+        assertEquals(1, value);
```

Comment:
Entweder JDoc oder eine Message:
```suggestion
        assertEquals("Skill should be executed exactly once" ,1, value);
```
als Beispiel